sky/templates/kubernetes-port-forward-proxy-command.sh.j2

```diff
@@ -13,6 +13,25 @@ if ! command -v nc > /dev/null; then
   exit
 fi
 
+# Checks if shuf is installed(Linux/macos)
+if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
We probably don't want to have separate checks for os and then check only for gshuf. For example, my mac machine has shuf available.

Can we simplify this check to:
1. Check if `shuf` is available
2. If not, check if `gshuf` is available
3. Else, raise error.